### ISSUE TITLE
Идентификация товаров

### ISSUE THREAD

**Issue by denyszamiatin** [denyszamiatin/crawler#11]
Реализовать возможность рассчета расстояния Ливенштейна между полученным названием на сайте и эталонным названием

**Comment by sergei4e** [comment]
А можно более подробно описать задачу и привести примеры?

Правильно понимаю, что у нас есть некий файл, где лежат эталонные названия, например:

Телевизор Samsung UE24H4070

И задача идентификации заключается в том, чтобы наш crawler понимал, что если на сайте есть названия:

Телевизор Samsung UE24H4070D
Телевизор Samsung UE 24 H 4070
Телевизор Samsung-UE24H4070D
Телевизор Sam Sung UE24H4070D
Телевизорsamsung UE24H4070D
Телевизор Самсунг UE24H4070
Телевизор-самсунг UE24H4070

то это все одно и то же и относил данные к эталонному названию:
"Телевизор Samsung UE24H4070"

Верно?

**Comment by denyszamiatin** [comment]
Да. Обычно для этого можно воспользоваться расстоянием Левенштейна

2015-10-12 23:04 GMT+03:00 Sergey Chernenko notifications@github.com:

**Comment by sergei4e** [comment]
Проверка одинаковых товаров будет осуществляться уже из базы данных, чтобы не сравнивать налету каждый товар со всеми остальными. Делаем отдельным параллельным процессом.
